gbp/git/repository.py

```diff
@@ -1450,11 +1450,14 @@ def rename_file(self, old, new):
 
 #{ Comitting
 
-    def _commit(self, msg, args=[], author_info=None):
-        extra_env = author_info.get_author_env() if author_info else None
+    def _commit(self, msg, args=[], author_info=None, committer_info=None):
+        extra_env = author_info.get_author_env() if author_info else {}
+        if committer_info:
```

Comment:
If we change from None to {} here we need to fixup _build_env to return None for extra_env being {} or None